vuu-ui/packages/vuu-layout/src/layout-persistence/RemoteLayoutPersistenceManager.ts

```diff
@@ -134,13 +138,49 @@ export class RemoteLayoutPersistenceManager
   }
 
   saveApplicationLayout(layout: LayoutJSON): Promise<void> {
-    // TODO POST api/layouts/application #71
-    console.log(layout);
-    return new Promise((resolve) => resolve());
+    return new Promise((resolve, reject) =>
+      fetch(`${baseURL}/${applicationLayoutsSaveLocation}`, {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+          "username": "vuu-user"
+        },
+        body: JSON.stringify(layout),
+      })
+      .then((response) => {
+        if (!response.ok) {
+          reject(new Error(response.statusText));
+        }
+        resolve();
+      })
+      .catch((error: Error) => {
+        reject(error);
+      })
+    );
   }
 
   loadApplicationLayout(): Promise<LayoutJSON> {
-    // TODO GET api/layouts/application #71
-    return new Promise((resolve) => resolve(defaultLayout));
+    return new Promise((resolve, reject) =>
+      fetch(`${baseURL}/${applicationLayoutsSaveLocation}`, {
+        method: "GET",
+        headers: {
+          "username": "vuu-user",
+        },
+      })
+      .then((response) => {
+        if (!response.ok) {
+          reject(new Error(response.statusText));
+        }
+        response.json().then((response: ApplicationLayout) => {
```

Comment:
I would name `response` something different here to differentiate from the `response` on line 170 and to make it clearer that thats the content of the response body rather than the whole response. Maybe `applicationLayout`?

Reply:
Good point, I didn't notice they were different. Renamed here: 14f493b.